cmd/dashboard.go

```diff
@@ -121,7 +121,11 @@ func runDashboardCmd(cmd *cobra.Command, args []string) {
 	serverListen := dashboardserver.ListenType(viper.GetString(constants.ArgDashboardListen))
 	error_helpers.FailOnError(serverListen.IsValid())
 
-	if err := utils.IsPortBindable(int(serverPort)); err != nil {
+	serverHost := string(serverListen)
+	if serverListen == dashboardserver.ListenTypeNetwork {
```

Comment:
Even though both modes work after testing (not sure why `local` works yet), I probably need to either:
* add a case to convert `local` into `localhost` or `127.0.0.1`
* modify `ArgDashboardListen` to also accept a hostname and/or IP address like was done for the dabase.

Reply:
Reserved the condition to pass `"127.0.0.1"` to `IsPortBindable` if server listen is `local`, `""` otherwise.